JS_HW_2/HW_JS_2_task_3.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        // Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+"use strict";
+const minAge = 18; 
+const maxAge = 60;
+let age;
+age = prompt("What's your age?",); 
+const numberedAge = Number(age); 
```

Comment:
```suggestion
const numberedAge = Number(age); 
```
Move this line to `else` block after `isNaN` check

Reply:
@VladimirLQA 
Thank you for reviewing.
I don't understand; please, can you explain me, where am I wrong? 
According to the last comment: I declared the variable (const numberedAge = Number(age);) before the *if* block for checking: if the converted to number type value is not equal to NaN. But if I move this variable to *else* block, the check in the *if* block doesn't know about the -> const numberedAge = Number(age);

Reply:
Firstly, you need to check value for isNaN then convert it, if the check passed
With this approach, we do not perform an unnecessary value conversion action
```js
if (isNaN(age)) {
    alert("Oops! The value is NaN. Please try again!");
} else {
const numberedAge = Number(age);
switch (true) {
```